access/src/main/java/dk/dbc/holdingsitems/HoldingsItemsDAO.java

```diff
@@ -143,6 +143,14 @@ public Set<String> getBibliographicIdsIncludingDecommissioned(int agencyId) {
                 .getResultList());
     }
 
+    public List<String> getHoldingItems(int agencyId) {
+        //noinspection unchecked
+        String query = "SELECT DISTINCT COALESCE(s.superseding, i.bibliographicRecordId) FROM ItemEntity i " +
+                "LEFT JOIN SupersedesEntity s ON i.bibliographicRecordId = s.superseded " +
+                "WHERE i.agencyId=:agencyId";
+        return em.createQuery(query, String.class).setParameter("agencyId", agencyId).getResultList();
+    }
+
```

Comment:
Den tager i vidst ikke højde for "discarded" status, og der skal træffes en bevids beslutning om "missing" også skal være med.

Reply:
Det slags havde den oprindelige ikke med i sine overvejelser, og den bare blev brugt til at dumpe, ved jeg ikke rigtigt hvad folk forventer.

Reply:
Så skulle der også være taget hånd om den slags